pkg/types/installconfig.go

```diff
@@ -106,20 +105,23 @@ func (p *Platform) Name() string {
 	return ""
 }
 
-// Networking defines the pod network provider in the cluster.
+// Networking defines cluster's network provider configuration.
 type Networking struct {
 	// Type is the network type to install
-	Type netopv1.NetworkType `json:"type"`
+	Type string `json:"type"`
 
 	// ServiceCIDR is the ip block from which to assign service IPs
 	ServiceCIDR ipnet.IPNet `json:"serviceCIDR"`
 
 	// ClusterNetworks is the IP address space from which to assign pod IPs.
-	ClusterNetworks []netopv1.ClusterNetwork `json:"clusterNetworks,omitempty"`
+	ClusterNetwork []ClusterNetworkEntry `json:"clusterNetworks,omitempty"`
```

Comment:
why is this not list of https://github.com/openshift/api/blob/master/config/v1/types_network.go#L58 ?

Reply:
Up to you. The cleanest would probably just be to use `configv1.NetworkSpec` directly. It depends on what you would like the installer to look like. Keep in mind that `NetworkSpec` may slowly change over time (additive only, of course).